src/core/server/rendering/__snapshots__/rendering_service.test.ts.snap

```diff
@@ -144,8 +144,8 @@ Object {
   "legacyMetadata": Object {
     "uiSettings": Object {
       "defaults": Object {
-        "registered": Object {
-          "name": "title",
+        "theme:darkMode": Object {
```

Comment:
A result of mocking `uiSettings.getRegistered` which is required to match the value of `uiSettings.getOverrideOrDefault` when reading defaults.